source/usage-examples/watch.txt

```diff
@@ -36,7 +36,11 @@ output:
    :copyable: false
 
    // results truncated
-   [{_id ObjectID("...")} {title Record of ...} {text No bytes, no problem....}]
+   {
+       "_id": ...,
+       "text": "Binary data, best to store with GridFS. Bucket that data",
+       "title": "Showcasing a Blossoming Binary"
+   }
```

Comment:
The output doesn't exactly match what the page says -- it tells the user that they need to run the Insert a Doc usage example, which is a different document.